### PR TITLE
Update routes.php (#120)

### PR DESCRIPTION
**Pull request by rudxde**
changing the tag of 'Lineare Algebra für Informatik' to the official one of campus online

### CODE REVIEW

**Comment by kordianbruck** [comment]
@rudxde its debateable what is the correct label for LinAlg - I've know it under that for the past five years :stuck_out_tongue:

File: redirect/routes.php [hunk 1]
```diff
@@ -260,6 +260,7 @@ class Route {
         'latex' =>  'sharelatex',
         'tex' =>  'sharelatex',
         'netz' => 'grnvs',
+        'la' => 'linalg',
```
Comment (kordianbruck) [inline redirect/routes.php]: You need to switch those two around: `'linalg' => 'la',`

Comment (rudxde) [reply redirect/routes.php]: i think, with the other changed line, it will makes sense 😉

Comment (kordianbruck) [reply redirect/routes.php]: No, your build/tests fail, just pointed out the reason why. Trust me :wink:

Comment (rudxde) [reply redirect/routes.php]: ah, ok.
then the arrow makes more sense 😅

**Comment by mammuth** [comment]
I'd propose keeping it as _linalg_ with the synonym for _la_.
@rudxde if you want this to be merged, please take a look at the requested changes and rebase on master.

**Comment by kordianbruck** [comment]
@rudxde @mammuth actually the problem is on this line: you will need to update the reference in the section to the entry - https://github.com/mammuth/TUM.sexy/blob/master/redirect/routes.php#L288

**Comment by kordianbruck** [comment]
@rudxde can we still expect a fix?

**Pull request merged**